docs/starters.yml

```diff
@@ -3033,3 +3033,20 @@
     - Includes plugins for offline support out of the box
     - TypeScript
     - Prettier & eslint to format & check the code
+- url: https://quietboy.net
+  repo: https://github.com/zhouyuexie/gatsby-starter-quiet
+  description: Gatsby out of the box blog, use TypeScript and highly customized style.
+  tags:
```

Comment:
we don't support all these tags. What do you think of the following?

```json
tags:
  - Typescript
  - Styling:SCSS
  - Linting
```

for a full list you can checkout https://www.gatsbyjs.org/starters/

Reply:
I am going to modify it.